tla-io/src/main/scala/at/forsyte/apalache/io/ConfigManager.scala

```diff
@@ -40,7 +44,8 @@ case class ApalacheConfig(
     runDir: Option[File] = None,
     configFile: Option[File] = None,
     writeIntermediate: Boolean = false,
-    profiling: Boolean = false)
+    profiling: Boolean = false,
+    features: Seq[Feature] = Seq())
```

Comment:
```suggestion
    enabledFeatures: Seq[Feature] = Seq())
```

Reply:
im ok with this change, but then I think we should change the cli flag to match, as we should keep those names in sync. What do you think?

Reply:
Good point, but I find `--enabled-features` a bit verbose. I'd say it's probably sufficiently clear by convention that `--features=a,b` enables features `a` and `b`; otherwise `--enable=a,b` might also be a good name.

Reply:
I was more worried that in this code snippet, the type signature alone does not tell us what parameter `features` is supposed to do.

Reply:
Good point! I added https://github.com/informalsystems/apalache/pull/1652/commits/aea03e8cc381009851553da656002bf883388921 hoping to address this without duplicating documentation :)